data/json/monsters/insect_spider.json

```diff
@@ -1857,5 +1857,356 @@
+    "fear_triggers": [ "FIRE" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [
+      [ "RANGED_PULL", 10 ],
+      [ "GRAB", 3 ],
+      [ "impale", 10 ],
+      {
+        "type": "bite",
+        "cooldown": 4,
+        "accuracy": 8,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_multiplier": 0.6 } ]
+      }
+    ],
+    "flags": [ "POISON", "SEES", "SMELLS", "IMMOBILE", "DIGS", "RANGED_ATTACKER" ]
+  },
+  {
+    "id": "mon_antlion_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant adult antlion" },
+    "description": "A ferocious mutant flying antlion as big as a cat, darting through the air with a cluster of fangs for a mouth.",
+    "default_faction": "antlion",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "volume": "30 L",
+    "weight": "40750 g",
+    "hp": 70,
+    "speed": 130,
+    "material": [ "iflesh" ],
+    "symbol": "y",
+    "color": "yellow",
+    "aggression": 20,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 1,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 8,
+    "armor_cut": 4,
+    "vision_day": 25,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_WEAK" ],
+    "fear_triggers": [ "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "SMELLS", "FLIES", "HEARS", "HIT_AND_RUN", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_strider_small",
+    "type": "MONSTER",
+    "name": { "str": "big water strider" },
+    "description": "This water bug is several times larger than a regular one.  It's moves are fast and fluid.",
+    "copy-from": "mon_strider_giant",
+    "proportional": { "hp": 0.5, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.5 },
+    "volume": "3 L",
+    "weight": "4 kg",
+    "melee_cut": 8,
+    "upgrades": { "half_life": 14, "into": "mon_strider_giant" }
+  },
+  {
+    "id": "mon_strider_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant water strider" },
+    "description": "This water glider is about as big as a domestic cat.  Its legs are more fin-like and a swollen body is mostly submerged underwater.",
+    "default_faction": "strider",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "9 L",
+    "weight": "12 kg",
+    "hp": 20,
+    "speed": 200,
+    "attack_cost": 200,
+    "material": [ "iflesh" ],
+    "symbol": "s",
+    "color": "light_gray",
+    "aggression": 10,
+    "morale": 30,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_cut": 3,
+    "dodge": 4,
+    "armor_bash": 1,
+    "armor_cut": 3,
+    "armor_bullet": 2,
+    "vision_day": 12,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_WEAK" ],
+    "fear_triggers": [ "FIRE", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": ["POISON", "SEES", "HEARS", "SMELLS", "SWIMS", "AQUATIC" ]
```

Comment:
`SWIMS` and speed 200 means they will be able to go 8 tiles' worth of water in a single turn, which is a bit overkill (and the same mistake that turned dragonfly naiads into my own little extinction event, so great minds and all that). An option would be to keep `SWIMS`, removing `AQUATIC` (they can walk on land, they just suck at it) and reducing their speed to 50 - they'll stay very fast in the water but struggle on land, another combo we don't really have.